test/peoplePicker.browser.js

```diff
@@ -75,39 +75,35 @@ test('People Picker', async t => {
   await t
     .click(editors.suggestedSeniorEditorSelection)
     .click(editors.peoplePickerOptions.nth(0))
-    .click(editors.peoplePickerOptions.nth(2))
-    .click(editors.peoplePickerOptions.nth(3))
-    .click(editors.peoplePickerOptions.nth(5))
-    .click(editors.peoplePickerOptions.nth(7))
-    .click(editors.peoplePickerInfo.nth(9))
+    .click(editors.peoplePickerOptions.nth(1))
 
     // open modal, and we should be able to select this editor as maximun limit is not reach
-    .expect(editors.peoplePickerModalErrorMaximum.count)
-    .eql(0)
-    .click(Selector('[data-test-id="accept"]'))
+    // .expect(editors.peoplePickerModalErrorMaximum.count)
```

Comment:
commented out test? :bomb: maybe it's just less data to go through because of the dummies

Reply:
We need to change the data passed to the dummy-api before this test will work again. I think we can include this as part of this PR as it should just be a container file change.

Reply:
I think this is the only things left to do now @will-byrne and I will be working on this today /cc @hdrury1

Reply:
tests have been reverted with the new api-dummy change